README.md

```diff
@@ -87,7 +117,7 @@ env LDFLAGS="-I/usr/local/opt/openssl/include -L/usr/local/opt/openssl/lib" pip
 
 _Please make sure there is no whitespace in the path to the cloned repository. SymmetricDS doesn't work with a path with whitespaces._
 
-To setup SymmetricsDs on your machine, click [here.](https://github.com/mercury-telemetry/mercury-telemetry/edit/master/README.md) In case you are on windows, please also refer to the wiki on the topic by going [here.](https://github.com/mercury-telemetry/mercury-telemetry/wiki/Setting-up-SymmetricDS-on-Windows) Windows support is not streamlined and is very error prone but this should be enough to get you started. In case you want to help us make this better please refer below as to how you can contribute to the project.
+To setup SymmetricDS on your machine, click [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/blob/master/symmetricds/README.md) In case you are on windows, please also refer to the wiki on the topic by going [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Setting-up-SymmetricDS-on-Windows) Windows support is not streamlined and is very error prone but this should be enough to get you started. In case you want to help us make this better please refer below as to how you can contribute to the project.
```

Comment:
URLs in this section are still the old repo, please update

Reply:
done